h2o-mapreduce-generic/src/main/java/water/hadoop/FSCloudingEventSource.java

```diff
@@ -0,0 +1,150 @@
+package water.hadoop;
```

Comment:
I think this file should be split into separate classes and maybe put into a package

Reply:
ok